src/tree_data.h

```diff
@@ -1440,6 +1440,23 @@ int lyd_lyb_data_length(const char *data);
  */
 void *lyd_set_private(const struct lyd_node *node, void *priv);
 
+/**
+ * @brief Get the data node based on a simple XPath.
+ * This API returns the closest parent of the node (or the node itself)
+ * identified by the nodeid (path). This API uses lyd_find_sibling() If cache is
+ * enabled and the siblings are NOT top-level nodes, this function finds the
+ * node in a constant time.
+ * @param[in] data_tree Existing data tree to get.
+ * @param[in] ctx Context to use.
+ * @param[in] path Simple data path (see @ref howtoxpath).
+ * @param[in] output possible values are 1 and 0. output value maps to pathoption.
+ * Only valid pathoption in this API context is LYD_PATH_OPT_OUTPUT.
+ * Pass 1 if pathotion LYD_PATH_OPT_OUTPUT to use, else pass 0.
```

Comment:
Please explain the option directly, you can basically the description of [the option](https://github.com/CESNET/libyang/blob/master/src/tree_data.h#L852).

Reply:
done